packages/xo-server/src/decorators.spec.js

```diff
@@ -8,7 +8,7 @@ describe('debounce()', () => {
   let i
 
   class Foo {
-    @debounce(1e1)
+    @debounce(10)
```

Comment:
Same difference, albeit clearer, I recon :smiley:

Reply:
at first glance I took it for an hex number, so I changed it.